_themes/launcher.md

```diff
@@ -0,0 +1,18 @@
+---
+heading: Launcher
+thumbnail: themes/images/thumbnails/launcher.jpg
```

Comment:
Thanks @smcdougall, Please add the screenshots 

![](https://assets-cdn.github.com/images/icons/emoji/unicode/1f604.png)